templates/index.html

```diff
@@ -2,14 +2,30 @@
 
 <h1 class="title">{{title}}</h1>
 
+<table>
+    <thead>
+        <tr>
+            <th><center>Organization Types&nbsp</center></th>
+            <th><center>&nbspNumber of Organizations</center></th>
+	</tr>
+	<tr>
+	    <th><br></th>
+	</tr>
+    </thead>
+    <tbody>
+        {% for org_type, display_name in org_types.items() %}
+            <tr>
+                <td><a href="/open-source-by-organizations/{{org_type}}">{{ display_name }}</a></td>
+                <td><center>{{ stats.by_type[org_type] }}</center></td>
+            </tr>
+            <tr>
+                <td colspan="2">&nbsp;</td> <!-- Add a row with an empty cell that spans both columns to create spacing between rows -->
+            </tr>
```

Comment:
I bet there is some way with Bulma to do this instead of additional table-row.

Reply:
Hope it's better